packages/react-components/src/components/utils/ChatMessageComponentUtils.tsx

```diff
@@ -97,7 +97,7 @@ const defaultOnRenderAttachmentDownloads = (
     <_AttachmentDownloadCards
       message={message as ChatMessage}
       /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
-      attachments={(message as ChatMessage).files || []}
+      attachments={(message as ChatMessage).attachments || []}
```

Comment:
Should this be an empty array? Maybe we should not show the `_AttachmentDownloadCards` if there are not attachments?

Reply:
thanks updated